src/Javis.jl

```diff
@@ -1,5 +1,6 @@
 module Javis
 
+using Animations
```

Comment:
I also think it might be wise to add in automatic exports of `Animations.jl` methods/objects similar to what we did for `Luxor.jl`. I don't like having to state at the beginning of Javis files `using Javis, Animations`. I just want to use Javis! :smile:

Reply:
Short recap of my reasoning:
I think we should acknowledge with `using Animations` that this is part of their package. In general I think it's nice to see for the user where stuff is coming from. Luxor is an exception as we overload the functionality so reexporting is our only option.

Reply:
Agreed!